CONTRIBUTING.md

```diff
@@ -59,3 +59,98 @@ tests.
 
 We have some static checks when you filing a PR or pushing commits to the project, please make sure you can pass all the tests and make sure the coding style meets our requirements.
 
+## Contribution guide
+
+### 1. Clone your fork and configure the upstream
+
+Firstly, you have to fork this repo: <https://github.com/cap-ntu/ML-Model-CI> and create a local clone of this fork
+
+```shell
+git clone git@github.com:<YOUR-USERNAME>/ML-Model-CI.git
```

Comment:
better use https://github.com/....

Reply:
> better use https://github.com/....

Did that mean cloning repo by HTTPS is more recommended than by SSH

Reply:
Users need to setup ssh. Windows does not come with ssh, so clone with HTTPS is more convenient